.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+version: 2
+jobs:
+  build:
+    docker:
+    - image: ubuntu:xenial
+    working_directory: ~/virtlet
+    steps:
+    - checkout
+    - setup_remote_docker
+
+    - run:
+        name: Ensure prerequisites
+        command: |
+          apt-get -qq update
+          apt-get install -y libxml2-utils jq curl ca-certificates openssh-client rsync socat wget
+
+    - run:
+        name: Install Docker client
+        command: |
+          set -x
+          VER="17.03.0-ce"
```

Comment:
Frankly I just copied CircleCI's official example verbatim. But yes, we may improve it.